ext_emconf.php

```diff
@@ -42,7 +50,8 @@
     'version' => '6.1.2',
     'constraints' => [
         'depends' => [
-            'typo3' => '10.4.0-11.5.99',
+            'php' => '7.4.0-8.3.99',
```

Comment:
Please change to `"php": "8.1 - 8.3"`

Reply:
Note that Debian testing (the next stable release) already comes with PHP 8.4 which works similar good as PHP 8.1 / 8.2 / 8.3, so that would also be possible. But as long as Kitodo.Presentation limits the PHP versions to 8.1 – 8.3, this is fine here, too. Local installations which want to use 8.4 simply have to patch each ext_emconf.php.